foundations/html_css/project/html-css-foundations-project.md

```diff
@@ -36,6 +36,8 @@ Once you have completed your project, feel free to look at the other solutions t
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
+Don't forget to commit early & often! You can reference the Commit Message lesson [here](https://www.theodinproject.com/paths/html_css/html-foundations/commit-messages.md)!
```

Comment:
All of these "here" links aren't ideal from an a11y standpoint per my previous comment about link text.

Reply:
Ah! In the other files 🤦🏼‍♀️ My bad. I'll update!